### PR TITLE
moving buttons to top on program list view

### LINKED ISSUE #843: Move New Program and Publish buttons to the top of the page for visibility purposes
Move New Program and Publish buttons to the top of the page for visibility purposes

### PR DESCRIPTION
**Pull request by gcapiel**
### Description
moving buttons to top on program list view

### Checklist
- [ ] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### Issue(s)
Fixes #843

### CODE REVIEW

File: universal-application-tool-0.0.1/universal-application-tool-0.0.1.iml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (hmhsu) [inline universal-application-tool-0.0.1/universal-application-tool-0.0.1.iml:1]: Did you mean to check this change in?

**Review by hmhsu — COMMENTED** [review]
Looks good to me. Please fix Java formatting by running `bin/fmt`.

File: universal-application-tool-0.0.1/universal-application-tool-0.0.1.iml (hunk 1 above)
Comment (gcapiel) [reply universal-application-tool-0.0.1/universal-application-tool-0.0.1.iml:1]: No

Comment (gcapiel) [reply universal-application-tool-0.0.1/universal-application-tool-0.0.1.iml:1]: No, I think that was added by IntelliJ

Comment (gcapiel) [reply universal-application-tool-0.0.1/universal-application-tool-0.0.1.iml:1]: No, it must have been put in by IntelliJ

Comment (hmhsu) [reply universal-application-tool-0.0.1/universal-application-tool-0.0.1.iml:1]: Would you remove it from the PR by running `git rm path/to/file`? You could also add it to `.gitignore` if you will.

Comment (gcapiel) [reply universal-application-tool-0.0.1/universal-application-tool-0.0.1.iml]: Done by using git rm

**Review by hmhsu — APPROVED** [review]

File: universal-application-tool-0.0.1/universal-application-tool-0.0.1.iml (hunk 1 above)
Comment (hmhsu) [reply universal-application-tool-0.0.1/universal-application-tool-0.0.1.iml]: Nice! Thanks.

**Pull request merged**